src/messages.rs

```diff
@@ -68,11 +68,11 @@ pub struct BuildScript {
     /// The package this build script execution belongs to
     pub package_id: PackageId,
     /// The libs to link
-    pub linked_libs: Vec<PathBuf>,
+    pub linked_libs: Vec<String>,
```

Comment:
I think this is ok to be `PathBuf`, as it's either a name (so relative path) or a full path. It can't be a not-pathlike name afaict

Reply:
Changed back.